src/main.cpp

```diff
@@ -193,18 +189,25 @@ po::options_description buildVisibleOptions()
         ("gbdxm-file,f", po::value<string>()->value_name("PATH"), "Input or output GBDXM file.");
 
     addShowOptions(desc);
-    addPackOptions(desc);
+    addPackOptions(desc, true);
     addUnpackOptions(desc);
 
     return move(desc);
 }
 
-po::options_description buildHiddenOptions()
+po::options_description buildParseOptions()
 {
     po::options_description desc;
     desc.add_options()
-        ("plaintext", "don't encrypt the model.")
-        ("image-type,i", po::value<string>()->value_name("TYPE"), "Image type. e.g. jpg (deprecated).");
+        ("verbose,v", "")
+        ("gbdxm-file,f", po::value<string>())
+        ("plaintext", "")
```

Comment:
You might leave the description even if it isn't shown with `help`.  It makes it clear what the option does.